inc/auth.class.php

```diff
@@ -138,33 +139,23 @@ static function getMenuContent() {
    function userExists($options=array()) {
       global $DB;
 
-      $query = "SELECT *
-                FROM `glpi_users`
-                LEFT JOIN `glpi_useremails` ON (`glpi_users`.`id` = `glpi_useremails`.`users_id`)
-                WHERE ";
-      $first = true;
-      foreach ($options as $key => $value) {
-         if ($first) {
-            $first = false;
-         } else {
-            $query .= " AND ";
-         }
-         $query.=" `$key`='$value'";
-      }
-
-      $result = $DB->query($query);
-      if ($DB->numrows($result) == 0) {
+      $result = $DB->request('glpi_users',
```

Comment:
According to the doc :
['AND' => [$options],
=> get each field on $options
=> using the keyword "AND"

For me, same as foreach($array as $val) { $req .= ' AND '  . $val; }